### DISCUSSION TITLE
Newman can not get environment variables when specify multiple files

### DISCUSSION THREAD

**Discussion by SpecialCareNa** [npm/feedback#152] [General]
`newman run test1.postman_collection.json -e tiansuo1.0.postman_environment.json -d role_get.csv role_post.csv role_put.csv role_delete.csv --export-environment /root/skyshuttle/csv/test/test.env`

**Comment by MylesBorins** [comment]
Hey @SpecialCareNa this discussion forum is for making suggestions about the npm service itself, I think you need to open an issue over at the [newman repo](https://github.com/postmanlabs/newman)
